tools/cldr-apps/src/org/unicode/cldr/web/DataSection.java

```diff
@@ -315,7 +315,14 @@ private String getPClass() {
                     if (inheritedLocale != null && XMLSource.CODE_FALLBACK_ID.equals(inheritedLocale.getBaseName())) {
                         return "fallback_code";
                     }
-                    if (inheritedLocale == CLDRLocale.ROOT) {
+                    /*
+                     * Formerly the following test was "if (inheritedLocale == CLDRLocale.ROOT)", but that was
+                     * unreliable due to more than one CLDRLocale object having the name "root".
+                     * Reference: https://unicode-org.atlassian.net/browse/CLDR-13134
+                     * Also: https://unicode-org.atlassian.net/browse/CLDR-13132
+                     * If and when CLDR-13132 is fixed, the former test with "==" might be OK here.
+                     */
+                    if (inheritedLocale != null && "root".equals(inheritedLocale.getBaseName())) {
```

Comment:
I'm not sure if this is a good idea, I think we should figure out why there is more than one object named root, since I thought the root isn't usually named "root"

we can give this a try i suppose.

Reply:
I completely agree we should solve the bigger "root" problem including [CLDR-13132] ASAP. In the meantime I'd like to fix this blue/red bug since it's fairly serious in its own right and it's easy to fix immediately.

This is what I've learned:
* CLDRLocale.ROOT.getBaseName() returns "root"
* ULocale.ROOT.getBaseName() returns "" (empty string)

For getPClass, inheritedLocale is a CLDRLocale, not a ULocale, so this change will work, even though evidently we sometimes do have multiple CLDRLocale objects (including CLDRLocale.ROOT) such that getBaseName() returns "root".